cve_bin_tool/cli.py

```diff
@@ -137,7 +137,8 @@ def scan_file(self, filename):
 
         # Ensure filename is a file
         if not os.path.isfile(filename):
-            raise InvalidFileError(filename)
+            self.logger.debug(f"Not a valid file type: {filename}")
+            return None
```

Comment:
If you take a look at the test suite, you'll see that your code can't be merged right now because we have a test expecting this error to be raised.  Raising an error is the right thing to do here, so I don't think we want to change the test, we want to fix this code.  

If you really want the debugger line, it could be placed above the raise, or done in the error handling for that raise.  Line 141 is not needed at all and should be removed.

Reply:
@terriko I proposed the change because I can predict that this could happen (there are lots of 'files' in Linux that aren't real files e.g. /dev/xxxx) rather than it being exceptional (this is what I promote with my engineers!)l. I think a value should be returned rather than an exception. Understand that it breaks the test suite so will remove the change.

Reply:
Honestly, you shouldn't be scanning "special" directories on linux because it's just a waste of cpu cycles the way things are set up, so I want people to be very clear that it's not working.  But you've reminded me that people probably *will* try to scan `/` and have this happen.  I think you already filed an issue mentioning it, can you expand why this was a problem for you there and maybe we can find a way to improve the error message in a way that's clear but more usable?